src/main/scala/exercises/06_Collections.scala

```diff
@@ -130,7 +136,10 @@ object Collections {
    * neat way using recursion. 
    */
   def tails[T](l: List[T]): List[List[T]] = {
-    error("fix me")
+    l match {
+      case List() => List(l)
```

Comment:
What's the difference between saying what you said in the first case and saying the following:

case List() => l 

?